### ISSUE TITLE
Host at UMass

### ISSUE THREAD

**Issue by dfaulken** [umts/umasstransit.rodeo#16]
For the future it'd be good to put this somewhere at UMass like all the rest of our apps.

**Comment by sherson** [comment]
Definitely. Will set something up for next year (I've been told we may have some AWS options soon).

**Comment by dfaulken** [comment]
Once we do this we can change [this line](https://github.com/umts/umasstransit.rodeo/blob/master/app/views/participants/welcome.haml#L31).

**Comment by dfaulken** [comment]
Our administrative coordinator sent out an email reminding us about this event. Have we arrived at a hosting solution?

**Comment by sherson** [comment]
The UMass AWS implementation got stalled, but I pinged UMassIT again on it. If that doesn't happen in time, will come up with another plan.
